app/scripts/Annotations2dTrack.js

```diff
@@ -365,7 +365,7 @@ class Annotations2dTrack extends TiledPixiTrack {
 
   click(graphics, viewPos, uid, event, payload) {
     this.select(graphics, viewPos, uid);
-    pubSub.publish('app.click', {
+    pubSub.publish('app.click', { // eslint-disable-line no-undef
```

Comment:
```suggestion
    this.pubSub.publish('app.click', {
```
No need to ignore this case